### DISCUSSION TITLE
Invert network: How to get the estimated deformation phase without accumulation?

### DISCUSSION THREAD

**Discussion by pizzawoo** [insarlab/MintPy#1183] [Q&A] [answered]
Hello everyone. This might be a trivial question. I'm currently using the `invert_network` to estimate the phase of each interferogram from a ifgramStack.h5 unwrapPhase_bridging with Least Square minimization on deformation phase. However the output in timeseries.h5 seems to be already accumulated. I've checked the code in [ifgram_inversion.py](https://github.com/insarlab/MintPy/blob/f3324b8bd611fdd24a769663db964a9f94812f20/src/mintpy/ifgram_inversion.py#L91) but I didn't see the accumulation operation. Is there any way to get the LSE deformation phase of each interferogram?

**Comment by yunjunz** [comment, marked as answer]
If you meant the sequential displacement between adjacent acquisitions, use `temporal_derivative.py timeseries.h5`.

**Reply by pizzawoo** [reply]
Thanks, that helps a lot! Actually I also applied a simple  **range2phase** operation to the output of the `temporal_derivative.py` to get the phase value.
